# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: accessible/windows/ia2/ia2Accessible.cpp
# Commit: ffa79d70878f
# Full Hash: ffa79d70878f731b772ecf7a6d26e24eb8eaa5cb
# Author: Trevor Saunders <tbsaunde@tbsaunde.org>
# Date: 2015-05-06 20:26:41
# Regressor Bug: 606080
# File Overlap Count: 1
# Description:
#   bug 606080 - on windows give accessibles a unique 32 bit id r=surkov
# ==============================================================================

diff -r 28c2b07d197c -r ffa79d70878f accessible/windows/ia2/ia2Accessible.cpp
--- a/accessible/windows/ia2/ia2Accessible.cpp	Mon Apr 13 18:30:07 2015 -0400
+++ b/accessible/windows/ia2/ia2Accessible.cpp	Tue Apr 14 12:45:43 2015 -0400
@@ -495,7 +495,7 @@
     return E_INVALIDARG;
 
   AccessibleWrap* acc = static_cast<AccessibleWrap*>(this);
-  *aUniqueID = - reinterpret_cast<intptr_t>(acc->UniqueID());
+  *aUniqueID = AccessibleWrap::GetChildIDFor(acc);
   return S_OK;
 
   A11Y_TRYBLOCK_END